.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "repo-infra"]
+	path = repo-infra
+	url = git@github.com:ideahitme/repo-infra.git
```

Comment:
I guess if you use the https url here, then you don't have to do all this custom sed and submodule stuff in the `.travis.yaml` file?